src/OrchardCore.Modules/OrchardCore.ContentFields/Services/DefaultContentPickerResultProvider.cs

```diff
@@ -48,15 +59,21 @@ public async Task<IEnumerable<ContentPickerResult>> Search(ContentPickerSearchCo
             var contentItems = await query.Take(50).ListAsync();
 
             var results = new List<ContentPickerResult>();
+            var settings = searchContext.PartFieldDefinition.GetSettings<ContentPickerFieldSettings>();
 
             foreach (var contentItem in contentItems)
             {
-                results.Add(new ContentPickerResult
+                var cultureAspect = await _contentManager.PopulateAspectAsync(contentItem, new CultureAspect());
+                using (CultureScope.Create(cultureAspect.Culture))
```

Comment:
use a semicolon `;` at the end of the using line and drop the `{  }` to save indentation. 
```
using CultureScope.Create(cultureAspect.Culture);
```

Reply:
@MikeKry this one please